.github/workflows/integration-tests.yml

```diff
@@ -6,8 +6,14 @@ on:
     branches: [main]
 
 jobs:
-  ollama:
+  test-matrix:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # Listing tests manually since some of them currently fail
+        # TODO: generate matrix list from tests/integration when fixed
+        test-type: [inference, datasetio, eval, inspect, post_training, providers, scoring]
```

Comment:
at least some of these are skipped, e.g. post-training, so this gives a wrong impression that we cover anything here. Perhaps drop these types from the list and add a TODO message to enable them once the tests are fixed?..

Reply:
OK we discussed it in private and it's probably not a big deal and hopefully we fix it in time so no one uses these "passing" as a concrete indication that integration tests ACTUALLY passed.

Reply:
what about the main thing namely -- `agents`?

Reply:
The poor agent is failing at the moment.

Reply:
Just added it to the CI to see the failure here.

Reply:
https://github.com/meta-llama/llama-stack/actions/runs/13911045721/job/38925157781?pr=1636